test_sets/Defects/Defects_release_8_0.txt

```diff
@@ -34,3 +34,4 @@
 ./test_scripts/Defects/8_0/3788_2_allow_RC_permission_for_App_after_PTU_with_REMOTE_CONTROL_appHMIType.lua
 ./test_scripts/Defects/8_0/3788_3_revoke_RC_permission_for_App_after_PTU_with_MEDIA_appHMIType.lua
 ./test_scripts/Defects/8_0/3797_3798_HMI_sends_SetAppProperties_and_tries_to_ActivateApp.lua
+./test_scripts/Defects/8_0/3805_crash_during_langchange.lua
```

Comment:
Probably should move this to `8_1` now

Reply:
I moved this script to the `8_1` folder and test set